### PR TITLE
http: adds test with chunked as a token

### PR DESCRIPTION
**Pull request by catenacyber**
## Ticket

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6415

### CODE REVIEW

File: tests/http-chunked/test.yaml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+args:
+  - -k none --set packet-alert-max=32
```
Comment (catenacyber) [inline tests/http-chunked/test.yaml:2]: This is always hard to debug

**Comment by victorjulien** [comment]
Why does it fail on 6.0.x?

**Comment by catenacyber** [comment]
> Why does it fail on 6.0.x?

Because `http.request_header` keyword was introduced in 7, doing another version

**Comment by catenacyber** [comment]
Replaced by #1642

**Pull request closed without merging**